app/bundles/CoreBundle/Tests/Twig/ThemesExtensionTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Mautic\CoreBundle\Tests\Twig;
+
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Twig\Extension\ThemesExtension;
```

Comment:
```suggestion
use Mautic\CoreBundle\Twig\Extension\ThemesExtension;
use PHPUnit\Framework\MockObject\MockObject;
```
You need to import the `MockObject` class; this should resolve the PHPStan errors.